objgraph.py

```diff
@@ -815,8 +834,10 @@ def gradient(start_color, end_color, depth, max_depth):
     return h, s, v
 
 
-def edge_label(source, target, shortnames=True):
-    if isinstance(target, dict) and target is getattr(source, '__dict__', None):
+def _edge_label(source, target, shortnames=True):
+    if isinstance(target, dict) and target is getattr(source,
+                                                      '__dict__',
+                                                      None):
```

Comment:
No.  Better a line longer than 80 characters than this.  Or

```python
if (isinstance(target, dict) and
        target is getattr(source, '__dict__', None)):
```
would also be acceptable.